modules/gitops-configuring-tls-for-redis-with-autotls-enabled.adoc

```diff
@@ -0,0 +1,107 @@
+// Module is included in the following assemblies:
+//
+// * /cicd/gitops/configuring-secure-communication-with-redis.adoc
+
+[id="gitops-configuring-tls-for-redis-with-autotls-enabled_{context}"]
+= Configuring TLS for Redis with autotls enabled
+
+[NOTE]
+====
+By default, the `autotls` setting is disabled.
+====
+
+.Procedure
+
+. Create an Argo CD instance with `autotls` enabled:
+
+.. In the *Administrator* perspective of the web console, use the left navigation panel to go to *Administration* -> *CustomResourceDefinitions*.
+
+.. Search for `argocds.argoproj.io` and click `ArgoCD` custom resource definition (CRD).
+
+.. Click *Instances* -> *Create ArgoCD*.
+
+.. In the YAML field, replace the YAML code:
++
+.Example Argo CD CR with autotls enabled
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd <1>
+  namespace: openshift-gitops <2>
+spec:
+  redis:
+    autotls: openshift <3>
+  ha:
+    enabled: true <4>
+----
+<1> Specify a name of the Argo CD instance. In this example, the name is `argocd`.
+<2> Specify a namespace where you want to run the Argo CD instance. In this example, the namespace is `openshift-gitops`.
+<3> This enables the `autotls` setting and creates a TLS certificate for Redis.
+<4> This enables the HA feature. If you do not want to enable HA, do not include this line or put the value as `enabled: false`.
++
+[NOTE]
+====
+Alternatively, you can enable `autotls` on an already existing Argo CD instance by running the following command:
+
+[source,terminal]
+----
+$ oc patch argocds.argoproj.io <instance-name> --type=merge -p '{"spec":{"redis":{"autotls":"openshift"}}}'
+----
+====
++
+[NOTE]
+====
+The setting with HA enabled requires a cluster with at least three worker nodes.
+====
+
+.. Click *Create*.
+
+.. Verify that the Argo CD pods are ready and running. For the Argo CD instance with HA enabled, this might take a few minutes.
```

Comment:
@ekristova Please follow the suggestion I gave above for a similar instance. I think it is for substep f of step 1 in the Configuring TLS for Redis with autotls disabled section for this portion of content.